ozark/pom.xml

```diff
@@ -7,12 +7,53 @@
         <artifactId>parent</artifactId>
         <version>1.0.0-m03-SNAPSHOT</version>
     </parent>
-    <artifactId>ozark</artifactId>
-    <packaging>jar</packaging>
-    <name>Ozark ${project.version} MVC RI</name>
+    <artifactId>javax.mvc</artifactId>
```

Comment:
I think we can leave it "ozark" for this version but for the next version upgrade it should be javax.mvc for the convention. Anyone has different knowledge for the convention ?